File: BackEnd/src/main/java/imi/spring/backend/models/UserDTO.java
package imi.spring.backend.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserDTO {
    protected String username;
    protected String email;
    protected String password;
}

File: FrontEnd/app/src/main/java/com/example/frontend/data/remote/dto/UserDTO.kt
package com.example.frontend.data.remote.dto

import com.example.frontend.domain.model.User

data class UserDTO(
    val id:Long,
    val email:String,
    val username:String,
    val password:String
)


fun UserDTO.toUser() : User{
    return User(
        id = id,
        email = email,
        username = username,
        password = password
    )
}
